src/types/customTypes.d.ts

```diff
@@ -8,3 +8,7 @@ export interface Forecast {
   weather: string;
   temp2m: Temperature;
 }
+
+export interface WeatherResponse {
+  dataseries: Forecast[];
+}
```

Comment:
See the Forecast interface just above that handles the JSON structure. This might have been unclear due to the way I was handling the DOM manipulation